### PR TITLE
feat(faucet): create faucet account phoenix chain

### LINKED ISSUE #724: Links to non compliant BPs return 404 page
**Describe the bug**
Links to non compliant BPs return 404 page

**To Reproduce**
1. Go to '[this page](https://mainnet.eosio.online/rewards-distribution)'
2. Click on 'non compliant bps'
3. Click on 'a bps name'

**Expected behavior**
A link to the BPs bp.json file should be included

**Screenshots**
image

### LINKED ISSUE #761: phoenix account creator

### PR DESCRIPTION
**Pull request by AngeloCG97**
## What does this PR have?

- Create faucet account phoenix chain
- Links in rewards distribution

Resolve #724
Resolve #761

### CODE REVIEW

File: hapi/src/utils/eos.util.js [hunk 1]
```diff
@@ -170,7 +170,7 @@ const getProducerSchedule = () => eosApi.getProducerSchedule({})
 const transact = async (actions, account, password) => {
   try {
     await walletUtil.unlock(account, password)
-  } catch (error) {}
```
Comment (leisterfrancisco) [inline hapi/src/utils/eos.util.js:173]: Is this space necessary?

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]

**Review by leisterfrancisco — APPROVED** [review]

**Comment by xavier506** [comment]
@AngeloCG97  do we need to define the `HAPI_EOS_FAUCET_ACCOUNT_PASSWORD` as a secret also?

File: kubernetes/configmap-dashboard.yaml [hunk 2]
```diff
@@ -87,6 +87,7 @@ data:
   HAPI_EOS_EXCHANGE_RATE_API: '${HAPI_EOS_EXCHANGE_RATE_API}'
   HAPI_COINGECKO_API_TOKEN_ID: '${HAPI_COINGECKO_API_TOKEN_ID}'
   HAPI_REWARDS_TOKEN: '${HAPI_REWARDS_TOKEN}'
+  HAPI_CREATE_ACCOUNT_ACTION_NAME: '${HAPI_CREATE_ACCOUNT_ACTION_NAME}'
```
Comment (xavier506) [inline kubernetes/configmap-dashboard.yaml:90]: @AngeloCG97 we are missing an env var here also

File: webapp/src/routes/BPJson.js [hunk 3]
```diff
@@ -44,14 +44,16 @@ const getBPJsonUrl = async (producer = {}) => {
...
+  } catch (error) { }
 
   return `${producerUrl}/${chainUrl}`.replace(/(?<=:\/\/.*)((\/\/))/, '/')
 }
 
 const getBpJSONOffChain = async (producer) => {
   try {
     const bpUrl = await getBPJsonUrl(producer)
+
+    if (bpUrl === 'http:///bp.json') return undefined
```
Comment (xavier506) [inline webapp/src/routes/BPJson.js:56]: how are we handling bp.json files on sites without SSL?

Comment (AngeloCG97) [reply webapp/src/routes/BPJson.js:56]: We make the request with http and not with https.

File: kubernetes/configmap-dashboard.yaml (hunk 2 above)
Comment (xavier506) [reply kubernetes/configmap-dashboard.yaml:90]: HAPI_EOS_FAUCET_ACCOUNT_PASSWORD ?

Comment (AngeloCG97) [reply kubernetes/configmap-dashboard.yaml:90]: Yes, it is
image

Comment (xavier506) [reply kubernetes/configmap-dashboard.yaml:90]: image
 Its in the github workflow but I  dont see it in the config map

File: webapp/src/routes/BPJson.js (hunk 3 above)
Comment (xavier506) [reply webapp/src/routes/BPJson.js:56]: image
[kubernetes/configmap-dashboard.yaml](https://github.com/eoscostarica/eosio-dashboard/pull/780/files/a5d7d2939478cf9940e1b430fea4cbeabc3aed5e#diff-7af4d6ed28e83b4e4e50624862513b116f45d4cda453c742cc4031997532602f)

**Pull request merged**